src/components/Forms/EditExpense.jsx

```diff
@@ -31,8 +31,11 @@ const EditExpenseModal = ({ isOpen, onClose, onDoneEdit, onChange, editedData })
                   value={editedData.category}
                   onChange={onChange}
                 >
-                  <MenuItem value="Transportation">Transportation</MenuItem>
                   <MenuItem value="Groceries">Groceries</MenuItem>
```

Comment:
Instead of making four separate calls to the 'MenueItem' component, you can store the parameter values Array. Then, you can iterate through this Array using the 'map()' function and call your child component just once.